app/components/MswBanner.tsx

```diff
@@ -59,7 +59,7 @@ export function MswBanner({ disableButton }: Props) {
         >
           Learn more <NextArrow12Icon />
         </button>
-      </label>
+      </aside>
```

Comment:
`jsx-a11y` got mad that this label didn't wrap a control. it was correct, it shouldn't be a label